### DISCUSSION TITLE
Grafana Candlestick Graph Support

### DISCUSSION THREAD

**Discussion by pbrissaud** [pbrissaud/suivi-bourse#34] [Ideas]
New Grafana 8.3 version add candlestick panel which can be useful in this project

**Additional context**
- [Grafana documentation](https://grafana.com/docs/grafana/latest/visualizations/candlestick/)

**Comment by pbrissaud** [comment]
Tested for the first time the new panel.

What do you think about it ? 

### Params of the panel :
![candlestick_params](https://user-images.githubusercontent.com/27202456/162593259-303835f9-68e8-455f-b6fc-5d8690cdb267.png)

### 7 days window : 
![candlestick_7days](https://user-images.githubusercontent.com/27202456/162593443-f25c1228-b0c7-4341-9459-9fa15d21bc80.png)

### 30 days window :
![candlestick_30days](https://user-images.githubusercontent.com/27202456/162593450-fab3336c-c507-4f26-86ef-645d122c316b.png)

**Comment by pbrissaud** [comment]
A nice webinar about stock monitoring on Grafana 

[https://grafana.com/go/grafanaconline/2022/grafana-cloud-stocks-and-crypto-monitoring/](https://grafana.com/go/grafanaconline/2022/grafana-cloud-stocks-and-crypto-monitoring/)

**Comment by mao-chong** [comment]
Hello Sir
I need some help for my project in python. I see your experience and i think you are very good at python
Can you contact me please?
Of course i will pay for your help thx
